include/zephyr/dt-bindings/misc/nordic-nrf-ficr-nrf54h20-eng0.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_MISC_NORDIC_NRF_FICR_NRF54H20_ENG0_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_MISC_NORDIC_NRF_FICR_NRF54H20_ENG0_H_
+
+#define NRF_FICR_TRIM_APPLICATION_HSFLL_TRIM_VSUP     0x3D8U
```

Comment:
MDK cannot be pulled into dt-bindings/, so it will need to be done this way. Note that other OSes like Linux have worked for decades without MDK, SVD etc. and have managed to support far more complex systems, arguably better than Zephyr.

Reply:
> MDK cannot be pulled into dt-bindings/, so it will need to be done this way.

I realize we can't include MDK headers this way, nor was I suggesting that.

> Note that other OSes like Linux have worked for decades without MDK, SVD etc. and have managed to support far more complex systems, arguably better than Zephyr.

Apologies for not being more clear, I'm not trying to say otherwise. For Nordic, redefining register offsets in separate files opens the door to misalignment against the MDK that its HAL aligns with. My comment here advises against this.

Reply:
I agree with @kyberdin - using hardcoded values in DT is not future-proof and should be avoided.

Reply:
FYI https://github.com/zephyrproject-rtos/hal_nordic/pull/161, so I guess we can close this topic